patches/minecraft/net/minecraft/block/BlockRedstoneWire.java.patch

```diff
@@ -13,7 +27,21 @@
          }
          else
          {
-@@ -393,35 +393,25 @@
+@@ -174,6 +176,13 @@
+             l = j;
+         }
+ 
++        if (i != j) 
++        {
++            net.minecraftforge.event.world.RedstoneUpdateEvent event = net.minecraftforge.event.ForgeEventFactory.onRedstoneUpdate(p_176345_1_, p_176345_2_, iblockstate1, i, j);
++            
++            j = event.getNewCurrent();
++        }
```

Comment:
Assigning j here does nothing, it is not used anywhere else in the function after this.